### ISSUE TITLE
setPlaybackParams() for adjusting playback rate

### ISSUE THREAD

**Issue by alierdogan7** [jeancsanchez/JCPlayer#80]
https://developer.android.com/reference/android/media/MediaPlayer#setPlaybackParams(android.media.PlaybackParams)

Hello,

In order to adjust the playback rate by using setPlaybackParams() function, I need to access MediaPlayer object. Is there a way to access it?

**Comment by jeancsanchez** [comment]
I think no because the MediaPlayer object is encapsulated on JcPlayerService class.

**Comment by alierdogan7** [comment]
Ok. I fixed the issue like the following:

I downloaded the project and added it as a module to my project. After being able to modify the JcPlayer's source code, I added a simple getMediaPlayer() getter method to JcPlayerManager class. After this I gained access to the MediaPlayer object in my own code and I became able to set playback rate with this line:

 
```
manager.getMediaPlayer().setPlaybackParams(manager.getMediaPlayer().getPlaybackParams().setSpeed(speed)) 
```
